### PR TITLE
QR Percentiles

### PR DESCRIPTION
**Pull request by WillQizza**
Building on top of @andy-mtng's existing code, the following pull request implements a ranking button on each profile that displays the profile's top QR estimate.

Solves #21

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -37,6 +37,7 @@ android {
             useJUnitPlatform()
         }
         animationsDisabled = true
+        unitTests.returnDefaultValues = true
```
Comment (jmmabanta) [inline app/build.gradle:40]:
```suggestion
```
remove this as this changes the behaviour of a lot of tests

Comment (jmmabanta) [reply app/build.gradle:40]: Nvm i recalled i time where i had this and it borked an existing test but now its not happening

File: app/src/main/res/layout/profile_activity.xml [hunk 2]
```diff
@@ -23,6 +23,15 @@
             app:layout_constraintTop_toTopOf="parent"
             app:srcCompat="@drawable/baseline_settings_24" />
 
+        <Button
```
Comment (jmmabanta) [inline app/src/main/res/layout/profile_activity.xml:26]: Can we replace this with a FAB with a 🏆 or a 🏅 or a podium icon instead?

File: app/src/test/java/com/cmput301w23t09/qrhunter/TestPhotoModel.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import android.graphics.Bitmap;
+import com.cmput301w23t09.qrhunter.locationphoto.LocationPhoto;
+import com.cmput301w23t09.qrhunter.player.Player;
+import java.util.ArrayList;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+public class TestPhotoModel {
```
Comment (jmmabanta) [inline app/src/test/java/com/cmput301w23t09/qrhunter/TestPhotoModel.java:15]:
```suggestion
public class TestLocationPhotoModel {
```

**Review by jmmabanta — APPROVED** [review]

**Review by IreneRSun — APPROVED** [review]

**Pull request merged**